loader/convert_data.go

```diff
@@ -56,6 +67,10 @@ func parseInsertStmt(sql []byte, table *tableInfo, columnMapping *cm.Mapping) ([
 			if sql[e] == '\n' && (sql[e-1] == ',' || sql[e-1] == ';') && sql[e-2] == ')' {
 				break
 			}
+			if sql[e] == '\n' && e-6 > s && bytes.Compare(sql[e-6:e], VALUES) == 0 {
```

Comment:
we have `e = s + 3` in L66, so we can use `sql[e-2]` in L67.
is it safe to use `sql[e-6:e]`?

Reply:
we have `e-6 > s` in this if condition, so it is safe to use `sql[e-6:e]`